frontend/src/app/commonComponents/RadioGroup.tsx

```diff
@@ -1,11 +1,15 @@
-import React from "react";
+import React, { ReactNode } from "react";
```

Comment:
Since the type below uses `React.ReactNode` this import isn't needed. You could also import it like is being done here, and then just use `ReactNode` above. Either is fine with me, I don't think we have a style for it.

Reply:
Thanks, yeah @nullflux helped me see this too and removed it in https://github.com/CDCgov/prime-simplereport/pull/869/commits/193818a977db1a3f91ea12707ba33f92d88c2fe5